omDomainJSON.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2023 The Superpower Institute Ltd.
+#
+# This file is part of OpenMethane.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Generate a JSON file describing the domain and grid.
+"""
+
+import numpy as np
+import netCDF4 as nc
+from omInputs import domainXr, domainProj
+from omOutputs import domainJSONOutputPath
+import json
+
+def makePoint(x, y):
+    return ([float(x), float(y)])
+
+def writeDomainJSON():
+    # Load raster land-use data
+    print("converting domain grid details to JSON")
+
+    domain = {
+        "crs": {
+            "projection_type": "lambert_conformal_conic",
+            "standard_parallel": float(domainXr.attrs['TRUELAT1']),
+            "standard_parallel_2": float(domainXr.attrs['TRUELAT2']),
+            "longitude_of_central_meridian": float(domainXr.attrs['STAND_LON']),
+            "latitude_of_projection_origin": float(domainXr.attrs['MOAD_CEN_LAT']),
+            "proj4": domainProj.to_proj4(),
+        },
+        "grid_properties": {
+            "rows": domainXr.sizes['ROW'],
```

Comment:
This might need to include a static reference to WGS84 coords, ie the bounds?

I think perhaps the projection centre coords are also the grid centre coords, and you can also derive the bounds from the `grid_cells` data, but perhaps it's best to include it here for convenience?